user_guide/admin_capabilities.rst

```diff
@@ -0,0 +1,461 @@
+
+
+.. _admin-capabilities-campaign-emails:
+
+Send Campaign Emails
+--------------------
+
+The capability enables users to :ref:`send emails within OroCRM email campaigns <user-guide-email-campaigns-send>`.
+This way, marketing associates can sed a personalized template-based letter to multiple users.
+
+The setting does not effect the user's ability to create and edit :ref:`email campaigns <user-guide-email-campaigns>` 
+and should only be allowed to employees authorized to send email campaigns. These may be emails to potential customers, 
+other system users, etc.  
+
+
+|
+
+.. image:: ./img/roles/email_campaign.png
+
+|
+
+
+.. _admin-capabilities-share-grid:
+
+Share grid view
+---------------
+
+The capability enables users to share the :ref:`grid <user-guide-ui-components-grids>` views that they have configured, 
+this way the user can :ref:`adjust a grid <user-guide-ui-components-grids-adjust>` and share it with other users.
+
+This is particularly useful for team-leads and heads of departments, who can share grids with their subordinates, but 
+may also be available to other users.
+
+|
+
+.. image:: ./img/roles/grid_share.png
+
+|
+
+ 
+.. _admin-capabilities-system-info:
+
+System Information
+------------------
+
+The capability enables users to see view the system information page. This page contains the list of Oro Packages and 
+third-party packages installed  and is usually used by the system administrators and integrators only.
+
+
+.. _admin-capabilities-system-config:
+
+System configuration
+--------------------
+
+The capability enables users to access the :ref:`system configuration page <admin-configuration>`. If it is set to 
+*"System"*, the system administrator can localize the system, change the display and tracking settings, and make other 
+important changes.
+
+
+.. _admin-capabilities-tags:
+
+Tag assign/unassign
+-------------------
+
+The capability enables users to assign/unassign :ref:`tags <user-guide-tags>` - non-hierarchical keywords or phrases 
+assigned to records. They provide additional information about records and are visible to all the system users.
+You can assign/unassign tags from the grid or from a View page of the record.
+
+Tags can be successfully utilized by all the users.
+
+
+.. _admin-capabilities-tags-all:
+
+Unassign all tags from entities
+-------------------------------
+
+The capability enables users to unassign tags added to the records by other users. The capability is only 
+meaningful if *"Tag assign/unassign"* has been enabled (set to *"System"*). If *"Tag assign/unassign"* has been enabled, 
+and *"Unassign all tags from entities"* is set to *"None"*, the users will only be able to unassign tags added by 
+themselves -  this way you can restrict users from deleting tags of other users. The capability is usually available to 
+team leads, heads of departments and managers..
```

Comment:
This capability lets users unassign tags that have been added to records by other users. It is only meaningful if *"Tag assign/unassign"* has been enabled (set to *"System"*). If *"Tag assign/unassign"* has been enabled, and *"Unassign all tags from entities"* is set to *"None,"* users will only be able to unassign tags that they have added themselves. This way, you can restrict users from deleting tags made by other users. This capability is usually available to team leads, department heads, and managers.